server/api/api/utils.py

```diff
@@ -1378,7 +1379,7 @@ async def delete_function(
                 return None
             except Exception as exc:
                 # return tuple with failure info
-                return function, str(exc)
```

Comment:
that might be intentional to avoid bloating the failure reason. @rokatyy correct?

Reply:
@liranbg yes, that's correct, I'd not change that

Reply:
noted. though I would like to move `_delete_nuclio_functions_in_batches` to `server/api/crud/runtimes/nuclio/function.py`. Its a more fitting plave IMO. wdyt?

Reply:
@alonmr agree 👍